script.js

```diff
@@ -0,0 +1,976 @@
+      standardKeyName: '.',
+      shiftKeyName: ',',
+    },
+  },
+  'keycode-38': {
+    getValue: false,
+    engKeyName: {
+      standardKeyName: '↑',
+      shiftKeyName: '↑',
+    },
+    rusKeyName: {
+      standardKeyName: '↑',
+      shiftKeyName: '↑',
+    },
+  },
+  'keycode-right-16': {
+    getValue: false,
+  },
+  'keycode-left-17': {
+    getValue: false,
+  },
+  'keycode-91': {
+    getValue: false,
+  },
+  'keycode-left-18': {
+    getValue: false,
+  },
+  'keycode-32': {
+    getValue: false,
+  },
+  'keycode-right-18': {
+    getValue: false,
+  },
+  'keycode-right-17': {
+    getValue: false,
+  },
+  'keycode-37': {
+    getValue: false,
+    engKeyName: {
+      standardKeyName: '←',
+      shiftKeyName: '←',
+    },
+    rusKeyName: {
+      standardKeyName: '←',
+      shiftKeyName: '←',
+    },
+  },
+  'keycode-40': {
+    getValue: false,
+    engKeyName: {
+      standardKeyName: '↓',
+      shiftKeyName: '↓',
+    },
+    rusKeyName: {
+      standardKeyName: '↓',
+      shiftKeyName: '↓',
+    },
+  },
+  'keycode-39': {
+    getValue: false,
+    engKeyName: {
+      standardKeyName: '→',
+      shiftKeyName: '→',
+    },
+    rusKeyName: {
+      standardKeyName: '→',
+      shiftKeyName: '→',
+    },
+  },
+  language: {
+    getValue: false,
+  },
+};
+
+class Keyboard {
+  constructor() {
+    this.main = null;
+    this.keysContainer = null;
+    this.keys = [];
+    this.language = null;
+    this.property = {
+      capsLock: {
+        press: false,
+        repeat: false,
+      },
+      shift: {
+        press: false,
+        repeat: false,
+      },
+      alt: {
+        press: false,
+        repeat: false,
+      },
+    };
+  }
+
+  init() {
+    this.main = document.createElement('div');
+    this.keysContainer = document.createElement('div');
+
+    this.main.classList.add('keyboard');
+    this.keysContainer.classList.add('keyboard__keys');
+    this.keysContainer.appendChild(this.createKeys());
+
+    this.keys = this.keysContainer.querySelectorAll('.keyboard__key');
+
+    this.main.appendChild(this.keysContainer);
+    document.body.prepend(this.main);
+  }
+
+  createKeys() {
+    try {
+      if (localStorage.getItem('language') === 'engKeyName' || localStorage.getItem('language') === 'rusKeyName') {
+        this.language = localStorage.getItem('language');
+      } else {
+        this.language = 'engKeyName';
+      }
+    } catch (err) {
+      this.language = 'engKeyName';
+    }
+
+    const fragment = document.createDocumentFragment();
+
+    Object.keys(keyDirectory).forEach((key) => {
+      const keyElement = document.createElement('button');
+
+      keyElement.setAttribute('type', 'button');
+      keyElement.setAttribute('data-keycode', key.replace(/[a-z -]/g, ''));
+      keyElement.setAttribute('data-get-value', keyDirectory[key].getValue);
+      keyElement.classList.add('keyboard__key');
+
+      function createButtonStyle(nameButton, ...className) {
```

Comment:
You can also declare this function on level up as 'createKeys'. In this place you can just call this function.

Reply:
Do as a class method? Roger

Reply:
Done